easybuild/easyconfigs/c/CDO/CDO-1.9.5-CrayGNU-18.07.eb

```diff
@@ -0,0 +1,36 @@
+# Updated by @gppezzi, @jgphpc, @omlins
+easyblock = 'ConfigureMake'
+
+name = 'CDO'
+version = '1.9.5'
+
+homepage = 'https://code.zmaw.de/projects/cdo'
+description = """CDO is a collection of command line Operators to manipulate
+and analyse Climate and NWP model Data."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.07'}
+toolchainopts = {'opt': True, 'pic': True}
+
+# Files visible at https://code.mpimet.mpg.de/projects/cdo/files, 
+# however link is different for each version!
+source_urls = ['https://code.mpimet.mpg.de/attachments/download/18264/'] 
+sources = [SOURCELOWER_TAR_GZ]
+checksums = [ '0c60f2c94dc5c76421ecf363153a5043' ]
+
+dependencies = [
+    ('cray-netcdf/4.6.1.2', EXTERNAL_MODULE),
+    ('cray-hdf5/1.10.2.0', EXTERNAL_MODULE),
+    # CDO: OpenMP support for compute intensive operators: https://code.zmaw.de/projects/cdo/wiki/OpenMP_support
+    # No MPI support
+]
+
+preconfigopts = 'export NC_CONFIG=$EBROOTNETCDF/bin/nc-config && '
```

Comment:
Change to `preconfigopts = 'export NC_CONFIG=`which nc-config` && '` since it nc-config is not found during the configuration. That's why we had the problems with netcdf4 support.

Reply:
If you look at the [metadata file](https://github.com/eth-cscs/production/blob/master/easybuild/cray_external_modules_metadata.cfg), you can see that we're defining the root variable to `NETCDF_DIR`, which contains only `lib` and `include` directories.

I would prefer that we use explicit paths when writing easyconfigs whenever possible, rather than using `which`. In this case you could use `$CRAY_NETCDF_DIR`